assets/js/converter.js

```diff
@@ -44,97 +45,99 @@ toMeasurement.addEventListener("change", function changeOutput(){
 });
 
 convertButton.addEventListener("click", function convert(){
-
-    switch(fromMeasurement.value) {
-        // WEIGHT CASES
-        case 'lb': {
-            if(toMeasurement.value === 'kg') {
-                resultMeasure.textContent = lbToKilo(convertibleMeasure.value);
-            } else if (toMeasurement.value === 't') {
-                resultMeasure.textContent = lbToTon(convertibleMeasure.value);
+    if (isNaN(parseInt(convertibleMeasure.value)) === true){
```

Comment:
Why this validation logic is not in the validation.js?

Reply:
I would create a validation function which validates and also replaces the , to . in case it's a number